### PR TITLE
Update free-courses-ar.md

### PR DESCRIPTION
**Pull request by JasonMa170699**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by JasonMa170699** [comment]
For resources
Description
Why is this valuable (or not)? yes it allows people to learn to code in Arabic
How do we know it's really free? It a free youtube video
For book lists, is it a book? For course lists, is it a course? etc course

**Comment by eshellman** [comment]
Be sure to check the free-programming-books-lint output for linter errors. Alphabetical order and spacing are the most common problems.

File: courses/free-courses-ar.md [hunk 1]
```diff
@@ -454,6 +454,7 @@
 ### Python
 
 * [تعلم أساسيات البرمجة](https://www.youtube.com/playlist?list=PLvGNfY-tFUN8HRLDE-D2sXvIgYwspdmFE) - غريب الشيخ \|\| Ghareeb Elshaik
+* [روس لغة بايثون - مقدمة ( بالعربي) - الدرس 1#Python in Arabic](https://www.youtube.com/watch?v=AA7lc2Y6bzI) - Hussam Hourani
```
Comment (eshellman) [inline courses/free-courses-ar.md:457]:
```suggestion* [قناة علم البيانات - حسام الحوراني](https://www.youtube.com/playlist?list=PLYW0LRZ3ePo6IYDS2K5IhmuP5qY3dmI9e) - Hussam Hourani
```

Comment (eshellman) [inline courses/free-courses-ar.md:457]:
```suggestion
* [قناة علم البيانات - حسام الحوراني](https://www.youtube.com/playlist?list=PLYW0LRZ3ePo6IYDS2K5IhmuP5qY3dmI9e) - Hussam Hourani
```

**Pull request merged**